src/config.js

```diff
@@ -12,7 +12,7 @@ const getConfig = () => {
 	return {
 		logging: true,
 		STREAMLABS_TOKEN: process.env['STREAMLABS_TOKEN'],
-		STREAMLABS_ENDPOINT: 'https://streamlabs.com/api/v1.0/alerts',
+		STREAMLABS_ENDPOINT: 'https://streamlabs.com/api/v1.0/alerts', // TODO: remove
```

Comment:
For me makes sense that we have that URL in the `StreamLabs` service, instead of the config.

Also, this URL is not going to change due to "on-premise" installations of `StreamLabs` since it's not possible to do so.